messaging-experiments.yaml

```diff
@@ -650,3 +650,89 @@
+                action:
+                  data:
+                    args: https://support.mozilla.org/en-US/kb/about-picture-picture-firefox
+                    where: tabshifted
+                  type: OPEN_URL
+                label:
+                  value: Learn More
+                  attributes:
+                    accesskey: L
+              secondary:
+                - action:
+                    type: CANCEL
+                  label:
+                    string_id: cfr-doorhanger-extension-cancel-button
+                - label:
+                    string_id: cfr-doorhanger-extension-never-show-recommendation
+                - action:
+                    data:
+                      category: general-cfrfeatures
+                    type: OPEN_PREFERENCES_PAGE
+                  label:
+                    string_id: cfr-doorhanger-extension-manage-settings-button
+            category: cfrFeatures
+            heading_text: Multitask with Picture-in-Picture
+            icon: chrome://global/skin/media/pictureinpicture.svg
+            info_icon:
+              label:
+                string_id: cfr-doorhanger-extension-sumo-link
+              sumo_path: extensionrecommendations
+            layout: icon_and_message
+            notification_text:
+              string_id: cfr-doorhanger-extension-notification
+            text: Pop video into a floating window to keep watching while you work in other tabs. Hover over any video and select the small blue rectangle that appears.
+          frequency:
+            lifetime: 3
+          targeting: 'true'
+          template: cfr_doorhanger
+          trigger:
+            id: openURL
+            params:
+              - youtube.com
+              - netflix.com
+              - vimeo.com
+              - twitch.com
+              - dailymotion.com
+              - hulu.com
+              - ustream.com
+              - metacafe.com
+              - espn.com
+              - disneyplus.com
+              - www.youtube.com
+              - www.netflix.com
+              - www.vimeo.com
+              - www.twitch.com
+              - www.dailymotion.com
+              - www.hulu.com
+              - www.ustream.com
+              - www.metacafe.com
+              - www.espn.com
+              - www.disneyplus.com
+    isHighVolume: false
+    userFacingName: Message Router Content Experiment Recommend Picture-in-Picture CFR
+    isEnrollmentPaused: false
+    experimentDocumentUrl: https://experimenter.services.mozilla.com/experiments/recommend-picture-in-picture-cfr/
+    userFacingDescription: If we introduce PiP via CFR for users visiting video sites frequently, then we will see more adoption of the PiP feature because we have observed the feature is not easily discovered via user research observations.
+  filter_expression: localeLanguageCode == 'en' && userPrefs.cfrFeatures && firefoxVersion == 77 && [userId, 'RECOMMEND_PICTURE_IN_PICTURE_CFR']|stableSample(0.05)
```

Comment:
This should be `targeting` since this is messaging system targeting

Reply:
This should be `en-US`

Reply:
This is missing `beta` channel

Reply:
Should use `bucketSample` with `n4s-workweek` as the namespace, starting at `0`

Reply:
not critical for English, but `localeLanguageCode == 'en'` doesn't look like the same thing as the `en-US` spec in Experimenter